### DISCUSSION TITLE
Welcome to randomstuff.py Discussions!

### DISCUSSION THREAD

**Discussion by izxxr** [nerdguyahmad/randomstuff.py#12] [Announcements]
## 👋 Welcome!
  We’re using GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by izxxr** [comment]
Neat.

**Reply by sachin-sankar** [reply]
Yea github discussions is always 
# Neat

**Reply by izxxr** [reply]
I'm still new to this though
